src/ActivityForm.css

```
@@ -0,0 +1,47 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+h1 {
+  background: #8b0000;
+  color: whitesmoke;
+  height: 80px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  width: 100%;
+  margin: 0;
+}
```

-  Replaced `whitesmoke` with its hex equivalent for consistency in color definitions.
- Using relative units like `rem` for `height` makes the layout more responsive and adaptable to different font sizes and screen resolutions. 

```suggestion
h1 {
  background: #8b0000;
  color: #F5F5F5;
  height: 5rem;
  display: flex;
  align-items: center;
  justify-content: center;
  text-align: center;
  width: 100%;
  margin: 0;
}
```